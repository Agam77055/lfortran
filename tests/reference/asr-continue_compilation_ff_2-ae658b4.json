{
    "basename": "asr-continue_compilation_ff_2-ae658b4",
    "cmd": "lfortran --fixed-form --show-asr --continue-compilation --no-color {infile} -o {outfile}",
    "infile": "tests/errors/continue_compilation_ff_2.f",
    "infile_hash": "885df7aadcb0e8e8478ccad6225f17e5c3cf699a04ad8372958afeab",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-continue_compilation_ff_2-ae658b4.stdout",
    "stdout_hash": "71a287d8618ef66129661b961ddc8f588efa8e08f6406cf0f2293259",
    "stderr": "asr-continue_compilation_ff_2-ae658b4.stderr",
    "stderr_hash": "133433dea59a2ba853a7854d1f6b11682d7133e7336c807148c0a39e",
    "returncode": 1
}