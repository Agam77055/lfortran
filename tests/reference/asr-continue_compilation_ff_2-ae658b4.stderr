tokenizer error: Expecting contains keyword before procedure definition
